modules/docs/src/04-codegen/01-customisation.md

```diff
@@ -134,3 +134,26 @@ whenever the `adtMember` trait is in use.
 Note: The `adtMember` trait has NO impact on the serialization/deserialization behaviors of smithy4s.
 The only thing it changes is what the generated code looks like. This is accomplished by keeping the
 rendered schemas equivalent, even if the case class is rendered in a different place.
+
+#### Specialized collection types
+
+Smithy supports `list` and `set`, Smithy4s renders that to `List[A]` and `Set[A]` respectively. You can also use the `@uniqueItems` annotation on `list` which is equivalent to `set`.
```

Comment:
Maybe add a note that uniqueItems is going away or remove that comment entirely?

Reply:
it is actually not going away. The set keyword is going away in favour of `uniqueItems` (aws has  pivoted on this)

Reply:
Ahh, I read that from the source and I thought it was the latest state